api.js

```diff
@@ -7,19 +7,26 @@ const api = axios.create({
 const getArticlesByCreateDate = () => {
   return api.get("articles?sort_by=created_at").then(({ data }) => {
     return data.articles;
+  }).catch((err) => {
+    return err
   });
 };
 
+const getArticles = (sortBy="created_at",order="asc") =>{
```

Comment:
As your article cards don't currently display any date, vote or comment count, it is difficult for the user to tell if the articles have actually been sorted by what they have selected - they can only tell that something has happened as the page has refreshed and there may be a different article at the top. I'd recommend adding some more info to your article cards to show this clearly

Reply:
@nc-saima-khanom do you mean I add it to the card on the home page with additional details?

Reply:
Yes exactly :)